Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM quay.io/konveyor/builder:ubi9-v1.20 AS builder
+FROM quay.io/konveyor/builder:ubi9-latest AS builder
```

Comment:
do we need any of Tiger's changes from here? https://github.com/openshift/oadp-operator/pull/1401/files

Reply:
very minor diff.. 1.22.3 vs 1.22.2. I think we agreed on 1.22.2 in some calls but im good with either.

Reply:
the ocp/release already pinned to 1.22.2 but should be close enough that it doesn't matter.